### ISSUE TITLE
Sounds no longer occur during keyboard input

### ISSUE THREAD

**Issue by jbphet** [phetsims/molarity#245]
While working on https://github.com/phetsims/sun/issues/697 I noticed that this sim no longer produces sound on every keyboard-nav-driven change of the slider values.  The live version *does* do this (version 1.5.6).  This may be related to recent work done on slider sound, or may be due to something else.  Regardless, it should be investigated and fixed.

**Comment by jbphet** [comment]
After a little research, I think this problem arose due to changes made in common code under https://github.com/phetsims/scenery/issues/997, where the pointer type `a11y` was changed to `pdom`.  I've changed the code in the Molarity sim to check for the `pdom` pointer type (see the commit linked above), but I'd like to get a quick thumbs up or down from the person who made the common-code change.  The reason I'm a little concerned is that I saw code that looked like the snippet below in `scenery.js`, and I want to make sure that I'm safe _only_ looking for the `pdom` value, and don't need to also check for `a11y`.

```js
  if ( name === 'a11y' || name === 'pdom' ) {
```

Assigning to @zepumph for what I think should be a very quick review.  Feel free to close if this if it looks good.
